From 68c6961c4b1640fc69804686a72503f30bf82012 Mon Sep 17 00:00:00 2001
From: Ronan Abhamon <ronan.abhamon@vates.fr>
Date: Thu, 6 Oct 2022 17:54:10 +0200
Subject: [PATCH 084/175] feat(linstorvolumemanager): support snaps when a host
 is offline

- Don't create diskless volumes during clone, delay it.

Signed-off-by: Ronan Abhamon <ronan.abhamon@vates.fr>
---
 drivers/linstorvolumemanager.py | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/linstorvolumemanager.py b/drivers/linstorvolumemanager.py
index e577f63..09aad42 100755
--- a/drivers/linstorvolumemanager.py
+++ b/drivers/linstorvolumemanager.py
@@ -1199,12 +1199,6 @@ class LinstorVolumeManager(object):
                 rsc_name=clone_volume_name,
                 storage_pool=self._group_name
             ))
-        for node_name in diskless_node_names:
-            resources.append(linstor.ResourceData(
-                node_name=node_name,
-                rsc_name=clone_volume_name,
-                diskless=True
-            ))
 
         # 5. Create resources!
         def clean():
-- 
2.45.2

